src/vaadin-radio-group.html

```diff
@@ -259,7 +258,9 @@
         }
 
         _addListeners() {
+    
           this.addEventListener('keydown', e => {
+            this.isKeyDown = true;
```

Comment:
Please do not use flags that look public. Any private flags should start with underscore, e.g. `_isKeyDown`